### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by EugeneButusov**
+ Feature #67 Game Center login for Profile

### CODE REVIEW

**Comment by EugeneButusov** [comment]
@borisspektor, it's ready to merge.

File: Soomla/CCUserProfileUtils.cpp [hunk 1]
```diff
@@ -31,6 +31,11 @@ namespace soomla {
              case TWITTER:
                  return __String::create("twitter");
                  break;
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_IOS)
```
Comment (borisspektor) [inline Soomla/CCUserProfileUtils.cpp]: @EugeneButusov I think we can remove the #if parts here, it just makes the code ugly

File: Soomla/CCUserProfileUtils.cpp [hunk 2]
```diff
@@ -48,6 +53,11 @@ namespace soomla {
         else if (actionTypeString->compare("twitter")) {
             return TWITTER;
         }
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_IOS)
```
Comment (borisspektor) [inline Soomla/CCUserProfileUtils.cpp]: @EugeneButusov and here

File: Soomla/CCUserProfileUtils.h [hunk 3]
```diff
@@ -32,6 +32,9 @@ namespace soomla {
         FACEBOOK = 0,
         GOOGLE = 2,
         TWITTER = 5
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_IOS)
```
Comment (borisspektor) [inline Soomla/CCUserProfileUtils.h]: @EugeneButusov and here

**Pull request merged**